ovs/lib/nodeinstallation.py

```diff
@@ -972,6 +972,12 @@ def _finalize_setup(client, node_name, node_type):
             storagerouter.rdma_capable = False
         storagerouter.node_type = node_type
         storagerouter.save()
+        if not Configuration.exists('/ovs/framework/edition'):
+            try:
+                val = storagerouter.features['alba']['edition']
+                Configuration.set(key='/ovs/framework/edition', value=val)
+            except TypeError:
```

Comment:
Type error seems weird here.
`storagerouter.features['alba']['edition']` could give off keys error
`Configuration.set` could throw NotFoundException, RuntimeError,...
better to use the default exception so except Exception: ...